content/docs/tutorials/add_advanced/congratulations@es.md

```diff
@@ -0,0 +1,30 @@
+---
+$title: ¡Felicitaciones!
+$order: 6
+---
+
+Ha terminado el tutorial "Añadir características avanzadas de AMP" y ha explorado con éxito muchos componentes avanzados clave de AMP.
+
+En este tutorial, aprendiste a:
+
+- Mostrar anuncios usando [amp-ad](/docs/reference/components/amp-ad.html)
+- Insertar videos de YouTube usando [amp-youtube](/docs/reference/components/amp-youtube.html)
+- Insertar Twitter cards usando [amp-twitter](/docs/reference/components/amp-twitter.html)
+- Incluir elementos responsivos de texto usando [amp-fit-text](/docs/reference/components/amp-fit-text.html)
+- Armar carruseles usando [amp-carousel](/docs/reference/components/amp-carousel.html)
+- Seguimiento de los patrones analíticos simples utilizando [amp-analytics](/docs/reference/components/amp-analytics.html)
+- Agregar navegación del sitio utilizando [amp-img](/docs/reference/components/amp-img.html), and [amp-sidebar](/docs/reference/components/amp-sidebar.html)
+- Utilizar fuentes personalizadas en AMP
+
+
+
+## Próximos pasos
+
+Para avanzar más en su experiencia de AMP, eche un vistazo a estos recursos:
+
+- [Crear páginas AMP interactivas](/es/docs/tutorials/interactivity.html): Aprenda cómo utilizar el amp-bind para crear páginas AMP interactivas
+- [Combina AMP con Aplicaciones Web Progresivas](/es/docs/guides/pwa-amp.html): Conozca cómo las páginas Web progresivas y las páginas de AMP se complementan entre sí
+- [AMP Start](https://www.ampstart.com/): Diseñe sus páginas con plantillas y componentes predefinidos
+- [AMP By Example](https://ampbyexample.com/): Aprender de muestras y demostraciones en vivo
+- [AMP Channel on YouTube](https://www.youtube.com/channel/UCXPBsjgKKG2HqsKBhWA4uQw): Aprenda viendo videos específicos de AMP
+- [AMP components](https://www.ampproject.org/docs/reference/components): Obtenga detalles sobre todos los componentes de AMP
```

Comment:
Change the link to `/es/docs/reference/components.html`

Reply:
Done